manager/integration/tests/test_basic.py

```diff
@@ -2051,3 +2053,30 @@ def test_expansion_canceling(clients, core_api, volume_name, pod):  # NOQA
     delete_and_wait_pod(core_api, pod_name)
     delete_and_wait_pvc(core_api, expansion_pvc_name)
     delete_and_wait_pv(core_api, expansion_pv_name)
+
+
+@pytest.mark.coretest  # NOQA
+@pytest.mark.skip(reason="TODO")
+def test_running_volume_with_scheduling_failure():
+    """
+    Test if the running volume still work fine
+    when there is a scheduling failed replica
+
+    1. Disable the soft anti-affinity setting.
+    2. Create a volume, then create the corresponding PV, PVC and Pod.
+    3. Wait for the pod running and the volume healthy.
+    4. Write data to the pod volume and get the md5sum.
+    5. Disable a node scheduling.
```

Comment:
Is. this same as "Disable scheduling on a node where a replica is running" ?

Reply:
Yes.